packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.spec.js

```diff
@@ -1,128 +1,107 @@
+        <Grommet
+          theme={zooTheme}
+        >
+          <ChoiceButton
+            choiceId='RDVRK'
+            choiceLabel='Aardvark'
+            disabled
+            onChoose={onChooseSpy}
+            tabIndex={0}
+          />
+        </Grommet>
+      )
+      const choiceButton = screen.getByRole('menuitemcheckbox', { name: 'Aardvark' })
+      await userEvent.click(choiceButton)
       expect(onChooseSpy).to.not.have.been.called()
-      onChooseSpy.resetHistory()
-    })
-  })
-
-  describe('when there is a small thumbnail', function () {
-    let media
-    before(function () {
-      wrapper.setProps({ thumbnailSize: 'small' })
-      media = wrapper.find(Media)
-    })
-
-    it('should render a thumbnail', function () {
-      expect(media).to.have.lengthOf(1)
-      expect(media.props().src).to.equal('cat.jpg')
-    })
-
-    it('should set the height of the thumbnail', function () {
-      expect(media.props().height).to.equal(21)
-    })
-
-    it('should calculate the width based on the height using set aspect ratio', function () {
-      expect(media.props().width).to.equal(Math.round(21 * THUMBNAIL_ASPECT_RATIO))
-    })
-  })
-
-  describe('when there is a medium thumbnail', function () {
-    let media
-    before(function () {
-      wrapper.setProps({ thumbnailSize: 'medium' })
-      media = wrapper.find(Media)
-    })
-
-    it('should render a thumbnail', function () {
-      expect(media).to.have.lengthOf(1)
-      expect(media.props().src).to.equal('cat.jpg')
-    })
-
-    it('should set the height of the thumbnail', function () {
-      expect(media.props().height).to.equal(42)
-    })
-
-    it('should calculate the width based on the height using set aspect ratio', function () {
-      expect(media.props().width).to.equal(Math.round(42 * THUMBNAIL_ASPECT_RATIO))
-    })
-  })
-
-  describe('when there is a large thumbnail', function () {
-    let media
-    before(function () {
-      wrapper.setProps({ thumbnailSize: 'large' })
-      media = wrapper.find(Media)
-    })
-
-    it('should render a thumbnail', function () {
-      expect(media).to.have.lengthOf(1)
-      expect(media.props().src).to.equal('cat.jpg')
-    })
-
-    it('should set the height of the thumbnail', function () {
-      expect(media.props().height).to.equal(84)
     })
 
-    it('should calculate the width based on the height using set aspect ratio', function () {
-      expect(media.props().width).to.equal(Math.round(84 * THUMBNAIL_ASPECT_RATIO))
+    it('should not call onKeyDown on keyDown of the button', async function () {
+      const onKeyDownSpy = sinon.spy()
+      render(
+        <Grommet
+          theme={zooTheme}
+        >
+          <ChoiceButton
+            choiceId='RDVRK'
+            choiceLabel='Aardvark'
+            disabled
+            hasFocus
+            onKeyDown={onKeyDownSpy}
+            tabIndex={0}
+          />
+        </Grommet>
+      )
+      await userEvent.keyboard('{enter}')
```

Comment:
I don't think this `userEvent` is pressing 'Enter' on the choice button. You might need to specify a target element for the event.

Reply:
You should probably also update your user event tests from v13 to v14, and use [`userEvent.setup()`](https://testing-library.com/docs/user-event/setup/) to create a user.